unittests/test_pipeline.py

```diff
@@ -791,3 +791,51 @@ def extract_perf(patt, tag):
         self.assertIn('v1', log_output)
         self.assertIn('v2', log_output)
         self.assertIn('v3', log_output)
+
+
+class TestContainerPlatformsTest(TestRegressionTest):
```

Comment:
By inheriting from `TestRegressionTest`, all its tests will run, which you don't want it.

Reply:
I see that you are using some functions of the `TestRegressionTest`, but we can figure it out how to do it nicely.